NBXplorer.Client/DerivationStrategy/IStrategy.cs

```diff
@@ -38,11 +41,18 @@ public Derivation GetDerivation(KeyPath keyPath)
 		}
 		public abstract Derivation GetDerivation();
 
-		protected abstract string StringValue
+		protected abstract string StringValueCore
```

Comment:
remove this refactoring, this is not needed anymore

Reply:
This one is needed in order to not make each derivation type have to do `StringValue => $"{StringValueCore}{GetSuffixOptionsString()}";`